rs/config/src/embedders.rs

```diff
@@ -46,8 +46,10 @@ const DEFAULT_WASMTIME_RAYON_COMPILATION_THREADS: usize = 10;
 const DEFAULT_PAGE_ALLOCATOR_THREADS: usize = 8;
 
 /// Sandbox process eviction ensures that the number of sandbox processes is
-/// always below this threshold.
-pub(crate) const DEFAULT_MAX_SANDBOX_COUNT: usize = 7_000;
+/// always below this threshold. Idle sandboxes should be using at most ~5MiB
+/// resident memory with the on-disk compilation cache, so 10,000 sandboxes
+/// shouldn't be more than 50 GiB.
+pub(crate) const DEFAULT_MAX_SANDBOX_COUNT: usize = 10_000;
```

Comment:
Should we also decrease the `DEFAULT_SANDBOX_PROCESS_RSS`?

Reply:
Makes sense. I can bump it to 70 GB in case we need a bit more.

Reply:
The naming is confusing... I suggest we should keep the `DEFAULT_MAX_SANDBOXES_RSS` at 50 GiB, but reduce `DEFAULT_SANDBOX_PROCESS_RSS` to more accurately reflect the new expected sandbox size. Does that make sense?

Reply:
Oh yeah,  I forgot about that part. I moved the max back to 50 GB and set the default to 5 MB.